lib/commands/generate.js

```diff
@@ -22,6 +22,7 @@ module.exports = Command.extend({
     { name: 'classic', type: Boolean, default: false, aliases: ['c'] },
     { name: 'dummy', type: Boolean, default: false, aliases: ['dum', 'id'] },
     { name: 'in-repo-addon', type: String, default: null, aliases: ['in-repo', 'ir'] },
+    { name: 'skip-lint-fix', type: Boolean, default: false, aliases: ['slf'] },
```

Comment:
Let's remove the alias (this is uncommon enough that I don't think forcing folks to use the longer but more descriptive `--skip-lint-fix` flag).

Also, what do you think about doing `--lint-fix` as the flag, then folks could do `--no-lint-fix` "for free"?

Reply:
Good suggestions, done 👍